test/unit_test/cb_las_spanner_test.cc

```diff
@@ -680,12 +687,10 @@ BOOST_AUTO_TEST_CASE(check_spanner_rejects_actions_that_are_linear_combinations_
       size_t encounters = 0;
       for (auto& a_s : preds)
       {
-        if (a_s.action == 1 && a_s.score != 0.f) { encounters++; }
-        if (a_s.action == 2 && a_s.score != 0.f) { encounters++; }
         if (a_s.action == 3 && a_s.score != 0.f) { encounters++; }
       }
 
-      BOOST_CHECK_EQUAL(encounters, 2);
+      BOOST_CHECK_EQUAL(encounters, 1);
```

Comment:
This is checking the fourth action (action_4 in the comment above) has non-zero score, right?
do we need to check other actions probabilities?